omnigan/trainer.py

```diff
@@ -905,10 +912,58 @@ def eval(self, num_threads=5, verbose=0):
         print("******************DONE EVALUATING*********************")
 
     def save(self):
-        pass
+        save_dir = Path(self.opts.output_path) / Path("checkpoints")
+        save_dir.mkdir(exist_ok=True)
+        save_path = Path(f"cpkt_epoch_{self.logger.epoch}.pth")
```

Comment:
oh actually coming back from further in the code, you do use `cpkt` instead of `ckpt` everywhere so I guess that's fine, you're redefining the field haha

Reply:
Haha no that was just a persistent typo. I'll change it